packages/contentstack-clone/package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "@contentstack/cli-cm-clone",
   "description": "Contentstack stack clone plugin",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "author": "Contentstack",
   "bugs": "https://github.com/rohitmishra209/cli-cm-clone/issues",
   "dependencies": {
-    "@contentstack/cli-cm-export": "^1.3.0",
-    "@contentstack/cli-cm-import": "^1.3.0",
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-cm-export": "^1.4.0",
+    "@contentstack/cli-cm-import": "^1.4.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/management": "^1.6.1",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K If we use @oclif/core from the utility package we can remove the dependency from package.json right.?

Reply:
@antonyagustine removed @oclif/core dependency from package.json